python/sbp/client/examples/pusher_client.py

```diff
@@ -65,13 +68,12 @@ def connect_it(data):
       push_client.connection.bind('pusher:connection_established', connect_it)
       push_client.connect()
 
-      handler.add_callback(push_it, [SBP_MSG_OBS, SBP_MSG_BASE_POS, SBP_MSG_OBS_DEP_A])
-
       try:
-        while True:
-          time.sleep(0.1)
+        for delta, time, msg in handler.filter(OBS_MSG_LIST):
```

Comment:
Should we kill the tuple usage here and above? Passing in 0's is super janky and using the tuples ends up being super inflexible. A newer iterator introduced a `metadata` [column](https://github.com/swift-nav/libsbp/blob/master/python/sbp/client/loggers/json_logger.py#L143), which then becomes a nightmare to thread around everywhere - potentially an envelope around the msg makes more sense to pass around? @mookerji ?